config.py

```diff
@@ -11,6 +11,34 @@
     'Qwen-14B-Chat-Int4'
 ]
 
+black_file_pattern_list = (
```

Comment:
在config.py中添加的black_file_pattern_list包含了一些常见的非源代码文件类型，但可能存在过度过滤的问题。例如，.txt和.json文件有时也包含重要的项目信息。

```suggestion
考虑将一些可能包含重要信息的文件类型（如.txt和.json）从黑名单中移除，或者提供一个白名单机制来更精确地控制哪些文件需要分析。